docs/python_docs/python/tutorials/deploy/inference/image_classification_jetson.md

```diff
@@ -0,0 +1,109 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Image Classication using pretrained ResNet-50 model on Jetson module
+
+This tutorial shows how to install MXNet v1.6 with Jetson support and use it to deploy a pre-trained MXNet model for image classification on a Jetson module.
+
+## What's in this tutorial?
+
+This tutorial shows how to:
+
+1. Install MXNet v1.6 along with its dependencies on a Jetson module (This tutorial has been tested on Jetson Xavier AGX and Jetson Nano modules)
+
+2. Deploy a pre-trained MXNet model for image classifcation on the module
+
+## Who's this tutorial for?
+
+This tutorial would benefit developers working on Jetson modules implementing deep learning applications. It assumes that readers have a Jetson module setup with Jetpack installed, are familiar with the Jetson working environment and are somewhat familiar with deep learning using MXNet.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+* A [Jetson module](https://developer.nvidia.com/embedded/develop/hardware) setup with [Jetpack 4.4](https://docs.nvidia.com/jetson/jetpack/release-notes/) installed using NVIDIA [SDK Manager](https://developer.nvidia.com/nvidia-sdk-manager)
+
+* An SSH connection to the module OR display and keyboard setup to directly open shell on the module
+
+* [Swapfile](https://help.ubuntu.com/community/SwapFaq) installed, especially on Jetson Nano for additional memory (increase memory if the inference script terminates with a `Killed` message)
+
+## Installing MXNet v1.6 with Jetson support
+
+We start by installing MXNet dependencies
+```bash
+sudo apt-get update
+sudo apt-get install -y git build-essential libopenblas-dev libopencv-dev python3-pip
+sudo pip3 install -U pip
+```
+
+Then we download and install MXNet v1.6 wheel with Jetson support
+```bash
+wget https://mxnet-public.s3.us-east-2.amazonaws.com/install/jetson/1.6.0/mxnet_cu102-1.6.0-py2.py3-none-linux_aarch64.whl
```

Comment:
> Projects MUST direct outsiders towards official releases rather than raw source repositories, nightly builds, snapshots, release candidates, or any other similar packages

http://www.apache.org/legal/release-policy.html#publication

It would be better to replace this with cross-compilation instructions based on the official ASF release (https://downloads.apache.org/incubator/mxnet/1.6.0/) to be compliant with ASF policy.
The wheel you download here is not ASF compliant (contains CUDA code) and must not be distributed by MXNet officially.
Adding this link as an optional part of the tutorial and putting a big disclaimer that this is not an ASF release and provided at user's own risk by a third party could be fine. We should open a ticket with Apache Legal team to reconfirm.

Reply:
@mseth10 - How do we proceed further? @leezu - Can you help with ticket to Apache ? Any recommendation here apart from adding disclaimer.

Reply:
Hi @sandeep-krishnamurthy , as per an offline discussion with @leezu , there is an ongoing discussion with Apache on this issue: https://issues.apache.org/jira/browse/LEGAL-515
To move ahead with this tutorial, we have decided to update the jetson installation doc on MXNet website and referring to that as the primary method in the tutorial. As an alternative, we'll provide users with the option to download the wheel with the disclaimer that the wheel is not an ASF release.

Reply:
I have updated the tutorial accordingly and created a PR to update Jetson installation docs #18485 with the Make instructions. I'll add cmake cross-compilation steps to the doc in a later PR.

Reply:
Bertrand from the Incubator suggested the following disclaimer:

> WARNING: the following links are provided for your convenience but
they point to packages that are *not* provided nor endorsed by the
Apache Software Foundation.
As such, they might contain software components with more restrictive
licenses than the Apache License and you'll need to decide whether
they are appropriate for your usage. Like all Apache Releases, the
official Apache MXNet (incubating) releases consist of source code
only and are found at <link>.

Reply:
Thank you @leezu 
@mseth10 - Let us get this warning note put in the tutorial, get review and approve from @leezu and move this forward please. thank you.

Reply:
> Like all Apache Releases, the
official Apache MXNet (incubating) releases consist of source code
only and are found at .

Found at https://downloads.apache.org/incubator/mxnet/ ? @leezu

Reply:
Made the changes. Can you please help merge @leezu